consistency-tests/__tests__/ref-forwarded.test.tsx

```diff
@@ -52,3 +53,16 @@ describe("Typography elements", () => {
         expect(ref.current).toBeInstanceOf(type);
     });
 });
+
+describe("View element", () => {
```

Comment:
**suggestion:** I think it would be good to at least include one of the other tags to see how it behaves in that case (and see how it is typed).

Reply:
Also good to include here some tests like the ones @kevinbarabash mentioned here.

https://khanacademy.slack.com/archives/C4PE1QM5Y/p1690409647322639?thread_ts=1690406084.915429&cid=C4PE1QM5Y